corehq/apps/reports/dispatcher.py

```diff
@@ -275,3 +275,11 @@ class QuestionTemplateDispatcher(ProjectReportDispatcher):
     def get_question_templates(self, domain, report_slug):
         question_templates = dict(self.get_reports(domain))
         return question_templates.get(report_slug, None)
+
+
+class DataExportInterfaceDispatcher(ProjectReportDispatcher):
+    prefix = 'data_export_interface'
+    map_name = 'EXPORT_DATA_INTERFACES'
+
+    def dispatch(self, request, *args, **kwargs):
+        return super(DataExportInterfaceDispatcher, self).dispatch(request, *args, **kwargs)
```

Comment:
What's with the no-op dispatch method?  Just to have here for decorating later?

Also, I'm curious why you decided to use a custom dispatcher instead of just using normal views and `urls.py`?  Does it provide additional context or something?